Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+extension Image {
+    /// The image used in the application.
+    enum AppImage {
+        /// An image of a white alarm symbol suitable for use in UI.
+        static let alarmSymbolWhite = Image("Alarm_symbol_white")
+        /// An image of a white bed symbol suitable for use in UI.
+        static let bedSymbolWhite = Image("Bed_symbol_white")
+        /// An image of a black exit symbol suitable for use in UI.
+        static let exitSymbolBlack = Image("Exit_symbol_black@3x")
+        /// A general background image suitable for use in UI.
+        static let generalBackgroundImage = Image("General_background_image")
+        /// An image of a purple home symbol suitable for use in UI.
+        static let homeSymbolPurple = Image("Home_symbol_purple")
+        /// An image of a purple moon symbol suitable for use in UI.
+        static let moonSymbolPurple = Image("Moon_symbol_purple")
+        /// An image of a colored moon symbol with stars suitable for use in UI.
+        static let moonWithStarsSymbolColored = Image("Moon_with_stars_symbol_colored")
+        /// An image of a purple moon symbol with stars suitable for use in UI.
+        static let moonWithStarsSymbolPurple = Image("Moon_with_stars_symbol_purple")
+        /// An onboarding background image suitable for use in UI.
+        static let onboardingBackgroundImage = Image("Onboarding_background_image")
+        /// A sleeping astronaut image suitable for use in UI.
+        static let sleepingAstronautImage = Image("Sleeping_astronaut_image")
+        /// An image of a statistic symbol suitable for use in UI.
+        static let statisticsSymbolPurple = Image("Statistics_symbol_purple")
+        /// An image of a colored sun symbol suitable for use in UI.
+        static let sunSymbolColored = Image("Sun_symbol_colored")
+        /// An image of a white symbol with a tick mark suitable for use in UI.
+        static let tickMarkSymbolWhite = Image("Tick-mark_symbol_white")
```

Comment:
nit: can we be consistent when naming assets?
```suggestion
        static let tickMarkSymbolWhite = Image("Tick_mark_symbol_white")
```

Reply:
I corrected this name.